### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by twittner**
Addresses https://github.com/libp2p/rust-libp2p/issues/593.

cc: @gnunicorn, @folsen

### CODE REVIEW

**Comment by tomaka** [comment]
The only question is whether the license change to Apache/MIT corresponds to what we want.

File: src/connection.rs [hunk 1]
```diff
@@ -72,6 +67,11 @@ where
         }
     }
 
+    /// Open a new outbound stream which is multiplexed over the existing connection.
```
Comment (dvdplm) [inline src/connection.rs]: Maybe s/Open/Opens/

**Comment by folsen** [comment]
> The only question is whether the license change to Apache/MIT corresponds to what we want.

MIT/Apache seems strictly better than pure MIT in my view. It also aligns better with Substrate which will be Apache-2.

**Comment by 5chdn** [comment]
Is there a reason to still maintain MIT?

**Comment by twittner** [comment]
I would argue similarly to this: https://www.rust-lang.org/en-US/faq.html#why-a-dual-mit-asl2-license

**Pull request merged**